### DISCUSSION TITLE
M2M tagging interface

### DISCUSSION THREAD

**Discussion by dimitrov-adrian** [directus/directus#3065] [Feature Requests]
Managing tags with directus is not very easy task. We have two options - to use tags which stores values in array (csv, json) or to use M2M (which provide slow way to input data).

It will be good to use tagging alike interface in M2M fields, so for interface setup will have an option to setup the referencing field (for example title) which will be used for autocomplete searches. And if "Allow others" option is set, then the new options will be inserted with the default values and "title" fields will be set to the input values.

**Comment by benhaynes** [comment]
A relational tags interface is very easy to build, and will be added. It's just not a priority right now since we're getting the core features in and stable. I expect in the next few weeks we'll be in a better position to start looking at new interfaces.

**Reply by egeozcan** [reply]
Perhaps a generic solution would be allowing inline search/create & add GUI control for all M2M related entities that require a single text field to be filled for new ones.

Also a special tag-like status can be added for such relations that, in search, they could turn into tag clouds and would require a single click to add to and remove from search.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
